css-canvas-medical/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <header class="row justify-between">
+      <img src="images/logo-blue.svg" alt="Canvas logo">
+      <nav class="row">
+        <a href="#product" class="nav-item">Product</a>
+        <a href="#implementation" class= "nav-item">Implementation</a>
+        <a href="#about-us" class="nav-item">About us</a>
+        <a href="#blog" class="nav-item">Blog</a>
+        <button class="blue-button">Contact Us</button>
+      </nav>
+    </header>
+    <div class="row hero-image">
+      <img src="images/hero.jpg">
+      <div class="hero-content relative">
+        <h1>The EMR Reimagined for Primary Care</h1>
+        <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Eveniet magni assumenda, dolores iure ipsam commodi hic, mollitia deleniti illo voluptatibus minima omnis facilis tenetur officiis fugiat doloremque nisi maiores minus accusantium ipsum ducimus? Ducimus, ex deleniti tenetur dicta perspiciatis fugiat ipsa quam magni expedita, sapiente omnis! Sit doloribus consequuntur labore!</p>
+        <button class="blue-button">Get Started</button>
+      </div>
+    </div>
+    <div class="row places">
+      <div class="column-full">
+        <h2>Clinicians Prefer Canvas</h2>
+        <p>Physician-owned primary care practices across the country have<br />already made Canvas the fabric of their care.</p>
+      </div>
+      <div class="row">
```

Comment:
This `row` shouldn't be necessary, you can likely have these `column-one-half` elements in the same `row` as the above `column-full` element, you just need to make sure the row can `wrap`